src/opening-period/description/OpeningPeriodDescription.tsx

```diff
@@ -21,31 +20,37 @@ export default function OpeningPeriodDescription({
   return (
     <>
       <h3 className="opening-period-section-title">Jakson kuvaus</h3>
-      <label htmlFor="openingPeriodTitle">Aukiolojakson otsikko *</label>
-      <TextInput
-        className="add-new-opening-period-title"
-        type="text"
-        name="openingPeriodTitle"
-        data-test="openingPeriodTitle"
-        id="openingPeriodTitle"
-        aria-invalid={errors.openingPeriodTitle ? 'true' : 'false'}
-        ref={register({ required: true, maxLength: 100 })}
-        helperText={
-          hasError ? 'Aukiolojakson otsikko on pakollinen' : undefined
-        }
-        invalid={hasError}
-      />
-      <label htmlFor="openingPeriodOptionalDescription">
-        Jakson valinnainen kuvaus
-      </label>
-      <TextArea
-        cols={90}
-        rows={9}
-        className="opening-period-optional-description"
-        id="openingPeriodOptionalDescription"
-        name="openingPeriodOptionalDescription"
-        ref={register({ maxLength: 255 })}
-      />
+      <div className="form-control">
```

Comment:
I think introducing this form-control helper div and class is actually a good idea. The margin-up was previously handled by styling on the labels, but this feels better and more right way to do it 👍

Reply:
I was thinking of separate component but then I was not sure where to but it.

Reply:
Maybe component/forms/Control.tsx ?

Reply:
Not sure if it needs a separate component. To me it's good already like this, and abstracting it to a separate component would just increase the amount of jumps reader have to do, without actually removing any code really.